stdlib/2and3/codecs.pyi

```diff
@@ -15,13 +15,18 @@ from abc import abstractmethod
 
 
 # TODO: this only satisfies the most common interface, where
-# str is the raw form and unicode is the cooked form.
+# bytes (py2 str) is the raw form and str (py2 unicode) is the cooked form.
 # In the long run, both should become template parameters maybe?
-# There *are* str->str and unicode->unicode encodings in the standard library.
-# And unlike python 3, they are in fairly widespread use.
+# There *are* bytes->bytes and str->str encodings in the standard library.
+# They are much more common in Python 2 than in Python 3.
+# Python 3.5 supposedly might change something there.
 
-_decoded = unicode
-_encoded = str
+if sys.version_info >= (3, 0):
+    _decoded = str
```

Comment:
Is this necessary? Could you just use "`Text`" for `_decoded`, and "`bytes`" for `_encoded`?